src/reducers/entities/channels.ts

```diff
@@ -521,6 +521,22 @@ function stats(state: RelationOneToOne<Channel, ChannelStats> = {}, action: Gene
 
 function groupsAssociatedToChannel(state: any = {}, action: GenericAction) {
     switch (action.type) {
+    case GroupTypes.RECEIVED_ALL_GROUPS_ASSOCIATED_TO_CHANNELS_IN_TEAM: {
+        const {groupsByChannelId} = action.data;
+        const nextState = {...state};
+
+        for (const channelID in groupsByChannelId) {
```

Comment:
Can we use a `for...of` loop instead of `for...in` and eliminate the `hasOwnProperty` check?